workflow-templates/im-deploy-az-function-manually-to-dev-subn.yml

```diff
@@ -182,7 +199,16 @@ jobs:
           deploy-status: ${{ steps.conclusion.conclusion }}
           timezone: 'america/denver'
 
-      # TODO: Remove if not sending Teams notifications
+      # TODO: Remove the following two steps if not sending Teams notifications
+      - name: Set the Azure Function Name
+        uses: im-open/set-variable-based-on-environment@v1.0.0
```

Comment:
Don't you already have the function name from above?

Reply:
This is in a different job.  So you have to set it twice which is stupid.

Reply:
Ahh, thanks for the explanation. That is stupid, but I guess understandable.